src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -344,7 +344,7 @@ private String getIdFromRequest(Request req) {
      * Get a request parameter value.
      * This method will halt the request if paramName is not provided in the request.
      */
-    private String getRequiredParamFromRequest(Request req, String paramName) {
+    protected String getRequiredParamFromRequest(Request req, String paramName) {
```

Comment:
A similar method also exists in`HttpUtils`.

Reply:
Updated to use the method it HttpUtils.